README.md

```diff
@@ -1 +1,8 @@
-# New-York-Times
\ No newline at end of file
+# New-York-Times
+This Project shows features of the Newyork Times.
+We are cloning the main features of the article.
+
+#### [Assignment link]( https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements)
+
```

Comment:
- Add a live preview of your page

Reply:
would we have to create the repository again? we didn't name the repository in a way it can be previewed live.

Reply:
- No, that's not necessary. Just insert a link of your html file in the [githack](https://raw.githack.com) and that should give you a nice preview.

Reply:
ok thanks